nnpdf_data/nnpdf_data/commondata/ATLAS_WPWM_7TEV_46FB/uncertainties.yaml

```
@@ -0,0 +1,3486 @@
+    description: 'Sys uncertainty idx: 107'
+    treatment: MULT
+    type: ATLASWZRAP11_1106
+  sys_corr_108:
+    description: 'Sys uncertainty idx: 108'
+    treatment: MULT
+    type: ATLASWZRAP11_1107
+  sys_corr_109:
+    description: 'Sys uncertainty idx: 109'
+    treatment: MULT
+    type: ATLASWZRAP11_1108
+  sys_corr_110:
+    description: 'Sys uncertainty idx: 110'
+    treatment: MULT
+    type: ATLASWZRAP11_1109
+  sys_corr_111:
+    description: 'Sys uncertainty idx: 111'
+    treatment: MULT
+    type: ATLASWZRAP11_1110
+  sys_corr_112:
+    description: 'Sys uncertainty idx: 112'
+    treatment: MULT
+    type: ATLASWZRAP11_1111
+  sys_corr_113:
+    description: 'Sys uncertainty idx: 113'
+    treatment: MULT
+    type: ATLASWZRAP11_1112
+  sys_corr_114:
+    description: 'Sys uncertainty idx: 114'
+    treatment: MULT
+    type: ATLASWZRAP11_1113
+  sys_corr_115:
+    description: 'Sys uncertainty idx: 115'
+    treatment: MULT
+    type: ATLASWZRAP11_1114
+  sys_corr_116:
+    description: 'Sys uncertainty idx: 116'
+    treatment: MULT
+    type: ATLASWZRAP11_1115
+  sys_corr_117:
+    description: 'Sys uncertainty idx: 117'
+    treatment: MULT
+    type: ATLASWZRAP11_1116
+  sys_corr_118:
+    description: 'Sys uncertainty idx: 118'
+    treatment: MULT
+    type: ATLASWZRAP11_1117
+  sys_corr_119:
+    description: 'Sys uncertainty idx: 119'
+    treatment: MULT
+    type: ATLASWZRAP11_1118
+  sys_corr_120:
+    description: 'Sys uncertainty idx: 120'
+    treatment: MULT
+    type: ATLASWZRAP11_1119
+  sys_corr_121:
+    description: 'Sys uncertainty idx: 121'
+    treatment: MULT
+    type: ATLASWZRAP11_1120
+  sys_corr_122:
+    description: 'Sys uncertainty idx: 122'
+    treatment: MULT
+    type: ATLASWZRAP11_1121
+  sys_corr_123:
+    description: 'Sys uncertainty idx: 123'
+    treatment: MULT
+    type: ATLASWZRAP11_1122
+  sys_corr_124:
+    description: 'Sys uncertainty idx: 124'
+    treatment: MULT
+    type: ATLASWZRAP11_1123
+  sys_corr_125:
+    description: 'Sys uncertainty idx: 125'
+    treatment: MULT
+    type: ATLASWZRAP11_1124
+  sys_corr_126:
+    description: 'Sys uncertainty idx: 126'
+    treatment: MULT
+    type: ATLASWZRAP11_1125
+  sys_corr_127:
+    description: 'Sys uncertainty idx: 127'
+    treatment: MULT
+    type: ATLASWZRAP11_1126
+  sys_corr_128:
+    description: 'Sys uncertainty idx: 128'
+    treatment: MULT
+    type: ATLASWZRAP11_1127
+  sys_corr_129:
+    description: 'Sys uncertainty idx: 129'
+    treatment: MULT
+    type: ATLASWZRAP11_1128
+  sys_corr_130:
+    description: 'Sys uncertainty idx: 130'
+    treatment: MULT
+    type: ATLASWZRAP11_1129
+  sys_corr_131:
+    description: 'Sys uncertainty idx: 131'
+    treatment: MULT
+    type: ATLASWZRAP11_1130
+  sys_corr_132:
+    description: 'Sys uncertainty idx: 132'
+    treatment: MULT
+    type: ATLASLUMI11
+  sys_corr_133:
+    description: 'Sys uncertainty idx: 133'
+    treatment: MULT
+    type: UNCORR.1
```

```suggestion
    type: UNCORR
```

I think this might be a typo...

---

I have just fixed this